### PR TITLE
documentation overhaul: quick pass cleanup

### PR DESCRIPTION
**Pull request by safl**
Initial removal of stale documentation and refactor / move of sections.

Depends on: https://github.com/xnvme/xnvme/pull/482 for fixing the docgen error

### CODE REVIEW

File: docs/backends/xnvme_info_block.out [hunk 1]
```diff
@@ -0,0 +1,43 @@
+
+
+
+
+
+
```
Comment (karlowich) [inline docs/backends/xnvme_info_block.out:6]: Is this whitespace intended?

File: docs/backends/xnvme_info_zoned.out [hunk 2]
```diff
@@ -0,0 +1,43 @@
+
+
+
+
+
+
```
Comment (karlowich) [inline docs/backends/xnvme_info_zoned.out:6]: Same here

**Review by karlowich — COMMENTED** [review]
A couple of comments

File: docs/backends/xnvme_info_block.out (hunk 1 above)
Comment (safl) [reply docs/backends/xnvme_info_block.out:6]: It is the raw output from the command in the ``.cmd`` file. If the commands output empty lines, then there will be empty lines here.

File: docs/backends/xnvme_info_zoned.out (hunk 2 above)
Comment (safl) [reply docs/backends/xnvme_info_zoned.out:6]: It is the raw output from the command in the ``.cmd`` file. If the commands output empty lines, then there will be empty lines here.

**Review by mbrsamsung — APPROVED** [review]
lgtm

**Pull request merged**